devices/YarpRobotLoggerDevice/include/BipedalLocomotion/YarpRobotLoggerDevice.h

```diff
@@ -177,6 +180,10 @@ class YarpRobotLoggerDevice : public yarp::dev::DeviceDriver,
     std::mutex m_bufferManagerMutex;
     robometry::BufferManager m_bufferManager;
 
+    const std::string m_rpcPortName{"/YarpLoggerDevice/commands/rpc:i"}; /**< name of Remote
```

Comment:
@S-Dafarra how does the multiple robot being teleoperated demo work? If the robot share the same YARP network, the RPC port name of each logger of each robot may conflict, perhaps we may want to add an option to set it?

fyi @GiulioRomualdi @LoreMoretti

Reply:
We never tried logging two robots at the same time

Reply:
Ack, but the two robot share the same network, right? In that case I guess it could make sense to have an option to set the name of the RPC port to avoid conflicts.

Reply:
> In that case I guess it could make sense to have an option to set the name of the RPC port to avoid conflicts.

Yes, I agree!

Reply:
Cannot I set it by default as `$YARP_ROBOT_NAME//YarpLoggerDevice/commands/rpc:i`?

Reply:
You would need to parse an env variable, and it might be empty. I think it is easier for the time being to simply read the port name from parameter

Reply:
I see. Then, maybe I can use [this already existing parameter](https://github.com/ami-iit/bipedal-locomotion-framework/blob/ba2a98aa93168374ece3f2913b15ee68bfaac6c4/devices/YarpRobotLoggerDevice/app/robots/ergoCubGazeboV1_1/yarp-robot-logger.xml#L11), which I have not yet figured out where it is being used.

Reply:
> I see. Then, maybe I can use [this already existing parameter](https://github.com/ami-iit/bipedal-locomotion-framework/blob/ba2a98aa93168374ece3f2913b15ee68bfaac6c4/devices/YarpRobotLoggerDevice/app/robots/ergoCubGazeboV1_1/yarp-robot-logger.xml#L11), which I have not yet figured out where it is being used.

Done with [ae1b791](https://github.com/ami-iit/bipedal-locomotion-framework/pull/915/commits/ae1b79103ce47107d0ff02ab8e4a1876d3fbee93)